### DISCUSSION TITLE
Error while installing Polypheny-UI on Ubuntu

### DISCUSSION THREAD

**Discussion by lehanshu** [polypheny/Polypheny-DB#387] [Q&A] [answered]
Hi Team,

I am getting below error while installing on local ubuntu using polyphony-control jar, can anyone help me with it?

FAILURE: Build failed with an exception.

What went wrong:
A problem occurred configuring root project 'polypheny-ui'.
> Could not resolve all files for configuration ':classpath'.
   > Could not resolve com.github.node-gradle:gradle-node-plugin:3.0.1.
     Required by:
         project :
      > Could not resolve com.github.node-gradle:gradle-node-plugin:3.0.1.
         > Could not get resource 'https://jcenter.bintray.com/com/github/node-gradle/gradle-node-plugin/3.0.1/gradle-node-plugin-3.0.1.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/github/node-gradle/gradle-node-plugin/3.0.1/gradle-node-plugin-3.0.1.pom'. Received status code 502 from server: Bad Gateway

**Comment by vogti** [comment, marked as answer]
Hi @lehanshu 
You are correct. JCenter (which has been shut down) is still listed as repository in the build script. Usually, this is not an issue. However, I suspect that there was a temporary issue with Maven Central that caused Gradle to fall back to JCenter. I have removed the JCenter dependency. Can you please try it again?

**Reply by lehanshu** [reply]
Hi @vogti 
Thanks for the prompt response and changes, it resolved the issue for me 👍
